src/styles/dictionary_concepts.scss

```diff
@@ -1,6 +1,20 @@
 .custom-dictionary-concepts {
   text-align: left;
   margin-top: -3%;
+  padding-right: 8%;
+  margin-left: 4%;
+}
+
+.bulk-concepts {
+  text-align: left;
+  margin-top: -3%;
+  padding-right: 4%;
+  margin-left: 5%;
```

Comment:
Is the margin applicable to all media?

Reply:
@Duncanian In this case I was just handling inconsistency of margin and padding in desktop layout.